onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -78,17 +78,20 @@ AllocatorPtr CUDAExecutionProvider::CreateCudaAllocator(OrtDevice::DeviceId devi
         },
         device_id,
         true,
-        {gpu_mem_limit,
-         static_cast<int>(arena_extend_strategy),
-         -1, -1});
+        {default_memory_arena_cfg ? default_memory_arena_cfg->max_mem : gpu_mem_limit,
+         default_memory_arena_cfg ? static_cast<int>(default_memory_arena_cfg->arena_extend_strategy) : static_cast<int>(arena_extend_strategy),
+         default_memory_arena_cfg ? default_memory_arena_cfg->initial_chunk_size_bytes : -1,
+         default_memory_arena_cfg ? default_memory_arena_cfg->max_dead_bytes_per_chunk : -1,
+         default_memory_arena_cfg ? default_memory_arena_cfg->initial_regrowth_chunk_size_bytes : -1});
 
```

Comment:
nit: could we reduce this to one ternary? 

```(default_memory_arena_cfg ? *default_memory_arena_cfg :  OrtArenaCfg{gpu_mem_limit, arena_extend_strategy, -1, -1, -1})```

Reply:
done